varya/README.md

```diff
@@ -0,0 +1,127 @@
+# Varya
+
+Varya is a styling system for quickly creating Gutenberg-ready themes for WordPress.
+
+## What does it do?
+
+When you reduce a theme design down to a set of systematic design decisions, you end up with something called a _Style Guide_. The Varya system works by taking the rules of a Style Guide and expressing them through carefully placed variables or _design tokens_ that influence the appearance of a WordPress site. It currently unifyies the overall of Gutenberg Blocks, the theme Header + Footer areas, WooCommerce, Jetpack and more. It also syncs styles between the editor and the frontend so that you don’t need to hand-write styles for both. This greatly speeds up the Gutenberg theme development process and reduces the amount of manual styling that typically goes into developing a theme. 
+
+### What controls does the system come with?
+
+  - **Fonts** - Font-family, size, weight, and line-height rules. 
+  - **Colors** - Primary, secondary, background, foreground and border colors. 
+  - **Spacing** - Sets an 8px vertical rhythm between all blocks and major components. Adds general spacing rules for blocks and components. Adds utility spacing classes for negative margins.
+  - **Responsive Logic** - Unifies the responsive behavior across various Blocks and Components to simplify behaviors.
+
+## How does it work?
+
+The system itself lives in the `/varya/sass` directory as a collection of Sass partials broken up by scope and hierarchy. The partials get compiled down to singular CSS files that live in the main Varya directory and cascade downward like so:
+
+**Frontend** 
+- `/varya/variables.css`
+- `/varya/style.css`
+
+**Editor** 
+- `/varya/variables-editor.css`
+- `/varya/style-editor.css`
+
+**Customizer**
+- `/varya/variables.css`
+- `/varya/style.css`
+
+In each view, the variables are loaded first and then the stylesheet is loaded which applies the variables.
+
+## How to use it
+
+To use the system, simply duplicate the `vayra-child` theme directory and rename it `my-theme-name`. You’ll also want to do a search for `varia-child` strings and replace them with `my-theme-name` as follows:
+
+  - Search for: `'varya-child'` and replace with: `'my-theme-name'` (with quotes).
+  - Search for: `varya_child_` and replace with: `my_theme_name_`.
+  - Search for: `Text Domain: varia-child` and replace with: `Text Domain: my-theme-name` in _style.css_.
+  - Search for:  `varia-child` and replace with: `my_theme_name`.
+  - Search for: `varia-child-` and replace with: `my-theme-name-`.
+
+Soon this process will be replaced by a `theme-dev-util` (similar to [this](https://github.com/Automattic/theme-dev-utils)) that allows you to run a command and automatically produce a child theme with all the strings already replaced.
+
+## Simple Child Theme Structure 
+(See: `/varya-child`)
```

Comment:
I may not understand fully, but isn't the child theme the actual theme you activate on your site? If so, should that be made more explicit?

Reply:
Yup, once I revise the “How To Use It” section this should become more clear.